app/models/user.rb

```diff
@@ -407,7 +407,11 @@ def onboarding
   def vet360_contact_info
     return nil unless VAProfile::Configuration::SETTINGS.contact_information.enabled && vet360_id.present?
 
-    @vet360_contact_info ||= VAProfileRedis::ContactInformation.for_user(self)
+    @vet360_contact_info ||= if Flipper.enabled?(:remove_pciu, self)
+                               VAProfileRedis::V2::ContactInformation.for_user(self)
+                             else
+                               VAProfileRedis::ContactInformation.for_user(self)
+                             end
   end
```

Comment:
@bosawt The only change to the user model is this. The previous code was removed. :)